utils/math.js

```diff
@@ -0,0 +1,25 @@
+// math.js
+
+// Function to generate a random integer within a specified range
+function getRandomInt(min, max) {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+// Function to check if a number is even
+function isEven(number) {
+  return number % 2 === 0;
+}
+
+// Function to check if a number is odd
```

Comment:
@gblotta-xoor Same as before, we should always use `JSDoc`

Reply:
@jaymdq https://github.com/jaymdq/code-review-process-workshop/pull/3/commits/03bee22f8f0e1f6ee0dbe0581a029051932d4ffc